Added further options

LepordCat opened the pull request:
Added BlockScreenCapture, CoverBoxedWindows, LingerExemptWnds, ForgetPassword, MsiInstallerExemptions, OpenClipboard, NtNamespaceIsolation, HideOtherBoxes, HideHostProcess, ProtectHostImages, AllowRawDiskRead and SeparateUserFolders.

Updated AllPages to link to the new pages and added links to pages which were not linked to AllPages.

LepordCat committed:
Create IsBlockCapture.md

LepordCat committed:
Update IsBlockCapture.md

LepordCat committed:
Update IsBlockCapture.md

LepordCat committed:
Update AllPages.md

LepordCat committed:
Create LingerExemptWnds.md

LepordCat committed:
Update LingerExemptWnds.md

LepordCat committed:
Update LingerExemptWnds.md

LepordCat committed:
Update AllPages.md

LepordCat committed:
Update IsBlockCapture.md

LepordCat committed:
Create ForgetPassword.md

LepordCat committed:
Create MsiInstallerExemptions.md

LepordCat committed:
Update AllPages.md

LepordCat committed:
Update AllPages.md

Added missing page links

LepordCat committed:
Update AllPages.md 

- Fixed link

LepordCat committed:
Update AllPages.md

-Fixed link

LepordCat committed:
Create OpenClipboard.md

LepordCat committed:
Update AllPages.md

LepordCat committed:
Create NtNamespaceIsolation.md

LepordCat committed:
Update AllPages.md

LepordCat committed:
Create HideOtherBoxes.md

LepordCat committed:
Update AllPages.md

LepordCat committed:
Create HideHostProcess.md

LepordCat committed:
Update AllPages.md

LepordCat committed:
Create ProtectHostImages.md

LepordCat committed:
Update AllPages.md

LepordCat committed:
Create AllowRawDiskRead.md

LepordCat committed:
Update AllPages.md

LepordCat committed:
Create SeparateUserFolders.md

LepordCat committed:
Update AllPages.md

isaak654 commented on Content/IsBlockCapture.md:
```diff
@@ -0,0 +1,13 @@
+# Is Block Capture
+
+_IsBlockCapture_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md). If enabled it will prevent sandboxed processes from accessing the images of the window outside the sandbox. For example:
```
```suggestion
_IsBlockCapture_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md). If enabled, it will prevent sandboxed processes from accessing the images of the window outside the sandbox. For example:
```

isaak654 commented on Content/ForgetPassword.md:
```diff
@@ -0,0 +1,17 @@
+# Forget Password
+
+_ForgetPassword_ is a global setting in [Sandboxie Ini](SandboxieIni.md). If set in [Sandboxie Control](SandboxieControl.md) the configuration password is cleared when the main window is hidden - and will need to be re-entered in order to modify configuration settings.
```
```suggestion
_ForgetPassword_ is a global setting in [Sandboxie Ini](SandboxieIni.md). If set in [Sandboxie Control](SandboxieControl.md) or [Sandman](PlusMigrationGuide.md), the configuration password is cleared when the main window is hidden - and will need to be re-entered in order to modify configuration settings.
```

Something similar has been done in PR #164

isaak654 reviewed (changes requested):
1. Please consider to document IsProtectScreen, as it's commonly confused with IsBlockCapture
2. Please add, wherever possible, in which version the settings were added ( for example OpenClipboard was introduced in version 0.7.5 / 5.49.8)

LepordCat committed:
Update Content/IsBlockCapture.md

Co-authored-by: isaak654 <isaak654@users.noreply.github.com>

LepordCat committed:
Update Content/ForgetPassword.md

Co-authored-by: isaak654 <isaak654@users.noreply.github.com>

LepordCat committed:
Update MsiInstallerExemptions.md

LepordCat committed:
Update OpenClipboard.md

LepordCat committed:
Update NtNamespaceIsolation.md

LepordCat committed:
Update HideOtherBoxes.md

LepordCat committed:
Update HideHostProcess.md

LepordCat committed:
Update ProtectHostImages.md

LepordCat committed:
Update AllowRawDiskRead.md

LepordCat committed:
Update SeparateUserFolders.md

LepordCat committed:
Update IsBlockCapture.md

LepordCat committed:
Update LingerExemptWnds.md

LepordCat committed:
Create IsProtectScreen.md

LepordCat committed:
Update AllPages.md

LepordCat committed:
Update IsBlockCapture.md

LepordCat committed:
Update README.md  

Corrected Typo

isaak654 commented on Content/MsiInstallerExemptions.md:
````diff
@@ -0,0 +1,15 @@
+   MsiInstallerExemptions=y
+```
+
+Use the 'MsiInstallerExemptions=y' option to allow MSIServer to run with a sandboxed system token and apply other exceptions. This option may help with installing a msi package.
````
```suggestion
Use the 'MsiInstallerExemptions=y' option to allow MSIServer to run with a sandboxed system token and apply other exceptions. This option may help with installing an MSI package.
```

The most common choice is to use "an MSI package" because "MSI" is typically pronounced as a single word.

isaak654 commented on Content/HideOtherBoxes.md:
```diff
@@ -0,0 +1,13 @@
+# Hide Other Boxes
+
+_HideOtherBoxes_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md) available since v0.3 / 5.42. If enabled, processes from other boxes will be hidden. By default Sandboxie enables this. Example of disabling this setting:
```
```suggestion
_HideOtherBoxes_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md) available since v0.3 / 5.42. By default, Sandboxie enables this feature, which allows processes to be hidden from other boxes. Example of disabling this setting:
```

isaak654 commented on Content/MsiInstallerExemptions.md:
```diff
@@ -0,0 +1,15 @@
+
+Use the 'MsiInstallerExemptions=y' option to allow MSIServer to run with a sandboxed system token and apply other exceptions. This option may help with installing a msi package.
+
+Related Sandboxie Plus setting: Sandbox Options > Security Options > Security Hardening> Allow MSIServer to run with a sandboxed system token and apply other exceptions if required.
```
```suggestion
Related Sandboxie Plus setting: Sandbox Options > Security Options > Security Hardening > Allow MSIServer to run with a sandboxed system token and apply other exceptions if required
```

isaak654 commented on Content/OpenClipboard.md:
```diff
@@ -0,0 +1,13 @@
+# Open Clipboard
+
+_OpenClipboard_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md) available since v0.7.5 / 5.49.8. It allows for clipboard access for a sandbox to be disabled. For example:
```
```suggestion
_OpenClipboard_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md) available since v0.7.5 / 5.49.8. It allows to disable clipboard access for a sandbox. For example:
```

isaak654 commented on Content/OpenClipboard.md:
````diff
@@ -0,0 +1,13 @@
+   OpenClipboard=n
+```
+
+Related Sandboxie Plus setting: Sandbox Options > General Options > Restrictions > Block read access to clipboard
````
```suggestion
Related Sandboxie Plus setting: Sandbox Options > General Options > Restrictions > Block read access to the clipboard
```

isaak654 commented on Content/ForgetPassword.md:
```diff
@@ -0,0 +1,17 @@
+
+See also: [Configuration Protection](ConfigurationProtection.md).
+
+Related Sandboxie Plus setting: Sandboxie Control > Options > Global Settings > Advanced Config > Sandboxie.ini Presets > Clear password when main window becomes hidden.
```
```suggestion
Related Sandboxie Plus setting: Options menu > Global Settings > Advanced Config > Sandboxie.ini Presets > Clear password when main window becomes hidden
```

isaak654 commented on Content/IsProtectScreen.md:
```diff
@@ -0,0 +1,15 @@
+
+A setting similar to _IsProtectScreen_ is [IsBlockCapture](IsBlockCapture.md).
+
+Related Sandboxie Plus setting: Sandbox Options > Security Options > Box Protection > Prevent processes from capturing window images from sandboxed windows.
```
```suggestion
Related Sandboxie Plus setting: Sandbox Options > Security Options > Box Protection > Prevent processes from capturing window images from sandboxed windows
```

LepordCat committed:
Update Content/MsiInstallerExemptions.md

Co-authored-by: isaak654 <isaak654@users.noreply.github.com>

LepordCat committed:
Update Content/HideOtherBoxes.md

Co-authored-by: isaak654 <isaak654@users.noreply.github.com>

LepordCat committed:
Update Content/MsiInstallerExemptions.md

Co-authored-by: isaak654 <isaak654@users.noreply.github.com>

LepordCat committed:
Update Content/OpenClipboard.md

Co-authored-by: isaak654 <isaak654@users.noreply.github.com>

LepordCat committed:
Update Content/OpenClipboard.md

Co-authored-by: isaak654 <isaak654@users.noreply.github.com>

LepordCat committed:
Update Content/ForgetPassword.md

Co-authored-by: isaak654 <isaak654@users.noreply.github.com>

LepordCat committed:
Update Content/IsProtectScreen.md

Co-authored-by: isaak654 <isaak654@users.noreply.github.com>

isaak654 committed:
Update option names

https://github.com/sandboxie-plus/sandboxie-docs/pull/162#issuecomment-2251105346
